tests/backstop_data/scenarios/activity-feed-panel.json

```diff
@@ -69,7 +69,7 @@
   },
   {
     "label": "Activity Feed Panel - On Contact Page",
-    "url": "{url}/civicrm/contact/search?reset=1",
+    "url": "{url}/civicrm/contact/view?cid={contactId:Betty Adams}&selectedChild=activity",
```

Comment:
We discussed about changing this to the Admin user. Please change.

Reply:
`{contactId:Betty Adams}`, the key is `contactId`, but we are sending a name, we should change this to `contactName`. Also cant we just send a string, instead of an object?

Reply:
> Also cant we just send a string, instead of an object
This is not an object but something used as a replacement patttern inside backstop while changing the contact name to respective id.
I agree that we should rename `contactId` to `contactName`.